clifford/test/test_numba_extensions.py

```diff
@@ -172,3 +172,19 @@ def test_pickling():
     e1._numba_type_  # int
     (e1 * 1.0)._numba_type_  # float
     assert pickle.loads(pickle.dumps(lt)) is lt
+
+
+def test_A_order():
+    import numpy as np
+
+    @numba.njit
+    def mul_mv(mv):
+        return mv*e3
+
+    mva = layout.MultiVector(np.ones(8)[::-1])
+    assert not mva.value.c_contiguous
```

Comment:
This assert fails because `Mv.__init__` always copies the input.

You can fix this with something awkward like
```
mva = layout.MultiVector(np.ones(8))
mva.value = mva.value[::-1]
```